iOS/SidePanels/build.cake

```diff
@@ -3,10 +3,19 @@
 
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
+var IOS_PODS = new List<string> {
```

Comment:
This library is not a binding project.  It is a port of the library JASidePanels.  You do not need to fetch the native libraries.